src/pages/Tutorials/Markdown/Universal-Deploy.md

```diff
@@ -44,33 +44,33 @@ If you're running into issues, check that your Python environment is properly se
 
 # Log into Prefect Cloud from the CLI
 
-To authenticate, you'll need to create a [Personal Access Token](/user/tokens) and configure it with the [Prefect Command Line Interface](https://docs.prefect.io/orchestration/concepts/cli.html#cli).
+To authenticate, you'll need to create an [API Key](/user/keys) and configure it with the [Prefect Command Line Interface](https://docs.prefect.io/orchestration/concepts/cli.html#cli).
 
-- In the hamburger menu in the top left corner go to **User -> Personal Access Tokens -> Create A Token**.
-- Copy the created token
-- Configure the CLI to use the access token by running
+- In the user menu in the top right corner go to **Account Settings -> API Keys -> Create An API Key**.
+- Copy the created key
+- Configure the CLI to use the key by running
```

Comment:
Can't we remove either the `prefect auth login` or creating a service account steps? I think these keys should be the same now, just scoped to different users.

Reply:
@znicholasbrown  We can remove the service account bit, and use the user's key to run the agent. I wasn't able to log in as a service account, which makes sense.
It seemed like we were considering service accounts a best practice for running agents though, is that not the case (alternatively, something we're not worried about for a getting started tutorial)?

Reply:
chatted in slack, decided to go with just the user key for simplicity's sake